documentation/peer-testing-1/peer-testing-report.md

```diff
@@ -28,31 +28,46 @@ In total, \_\_\_ individuals took part in our peer testing sessions. The members
 
 #### High
 
+- The default option of timezone field cannot be selected.
+- There should be a confirm message at each important button-clicking like clicking on submit buttons.
+- In the list of mentor's task, there is one that asks participants to find help page. A help page or FAQ should be built.
```

Comment:
I think this may have just been a discrepancy between the wording in the list of tasks that were displayed and the 'Mentor Tutorial'.